lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -488,7 +505,9 @@ private function buildToOneAssociationMappings(ClassMetadataInfo $metadata)
      *
      * @param \Doctrine\DBAL\Schema\Table $table
      *
-     * @return array
+     * @return ForeignKeyConstraint[]
+     *
+     * @psalm-return array<array-key, ForeignKeyConstraint>
```

Comment:
Not really needed, isn't it?

Reply:
Should be `array<string, ForeignKeyConstraint>`, since it returns the column name as keys IIRC.